web/src/lib/components/shared-components/upload-panel.svelte

```diff
@@ -65,12 +64,22 @@
         <div class="place-item-center mb-4 flex justify-between">
           <div class="flex flex-col gap-1">
             <p class="immich-form-label text-xm">
-              Remaining {$remainingUploads} - Processed {$successCounter + $errorCounter}/{$totalUploadCounter}
+              {$t('upload_progress', {
+                values: {
+                  remaining: $remainingUploads,
+                  processed: $successCounter + $errorCounter,
+                  total: $totalUploadCounter,
+                },
+              })}
             </p>
             <p class="immich-form-label text-xs">
-              Uploaded <span class="text-immich-success">{$successCounter}</span> - Error
-              <span class="text-immich-error">{$errorCounter}</span>
-              - Duplicates <span class="text-immich-warning">{$duplicateCounter}</span>
+              {@html $t('upload_status', {
```

Comment:
This leaves us vulnerable to XSS attacks. I don't have a solution for this yet, but it's also an issue in other places.

Reply:
Working on a solution, currently seems to be simplest named parametr replace within translated string.